pom.xml

```diff
@@ -90,6 +90,11 @@
       <artifactId>java-deptools-native</artifactId>
       <version>1.0.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-tree</artifactId>
```

Comment:
I don't see asm-tree being used in the code.
ClassVisitor and ClassReader are part of core ASM API - artifactId should be just "asm".

Reply:
Used in [Main.java](https://github.com/fedora-java/javapackages-validator/blob/8bdb81c6ae6e28d0695728c9dd4706e0647d2b12/src/main/java/org/fedoraproject/javapackages/validator/Main.java#L142) in order to correctly obtain the fully qualified name if class uri / filepath is given on input. This is later required by the custom `ClassLoader`.

Reply:
You are right, I will fix it.